test/datalog_tests/lb.dl

```diff
@@ -0,0 +1,17 @@
+typedef Payload = usize
+
+input relation Data(dat: Payload)
+input relation Workers(id: D3logLocationId)
+
+relation AllWorkers(workers: Vec<D3logLocationId>)
+
+AllWorkers(workers) :-
+    Workers(worker_id),
+    var workers = worker_id.group_by(()).to_vec().
+
+relation BalancedData(dat: Payload)
+
+BalancedData(dat) @worker :-
+    Data(dat),
+    AllWorkers(workers),
+    var worker = workers.nth(dat.hash64() % workers.len()).unwrap_or_default().
```

Comment:
Is there a way to get the hash64 for other data types?

Reply:
It should work for any type.